src/Helpers.jl

```diff
@@ -41,3 +41,10 @@ function new_vtk_step(Ω,file,hn,un,wn)
             cellfields=["hn"=>hn, "un"=>un, "wn"=>wn],
             nsubcells=4)
 end
+
+function new_vtk_step_tswe(Ω,file,hn,un,wn,en)
```

Comment:
I think we can do better here. I would try to stick into a single `new_vtk_step` function. 

To this end, you can modify the current signature to (Omega, file, cellfields) and pass directly, e.g., `["hn"=>hn, "un"=>un, "wn"=>wn]` as an argument to the last parameter.

Reply:
Good idea, I've modified the interface of the `new_vtk_step` as suggested